dockerfiles/init/modules/codenvy/templates/codenvy.env.erb

```diff
@@ -1,3 +1,11 @@
+# Include all custom CODENVY_* properties that user may has defined in its codenvy.env file
+<% ENV.each do |k,v| -%>
+<% if k.include? "CODENVY_" then  -%>
```

Comment:
In codenvy we configure everything with properties, I think this is bad idea to use props and ENV vars in the same time.